model/pipeline/src/main/proto/org/apache/beam/model/pipeline/v1/beam_runner_api.proto

```diff
@@ -365,6 +365,9 @@ message StandardPTransforms {
     // Represents the GroupIntoBatches.WithShardedKey operation.
     // Payload: GroupIntoBatchesPayload
     GROUP_INTO_BATCHES_WITH_SHARDED_KEY = 6 [(beam_urn) = "beam:transform:group_into_batches_with_sharded_key:v1"];
+
+    // Used for pubsub dynamic destinations.
+    PUBSUB_WRITE_V2 = 7 [(beam_urn) = "beam:transform:pubsub_write:v2"];
```

Comment:
move next to PUBSUB_WRITE?
can you add Payload: comment similar to others?

Reply:
Thanks, but I realized I was more interested in what the input to the composite transform is.
See comment for MERGE_WINDOWS
That I believe would show the difference between v1/v2 because IIRC the v2 that accepts dynamic destinations, each input is a KV or something that contains the destination topic, where v1 may just be accepting pubsub messages. And if my understanding is incorrect, a comment on the expected input would fix that :)